package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "commercetools-sunrise-data",
+  "version": "1.0.0",
+  "description": "[![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)",
+  "main": "index.js",
+  "scripts": {
+    "set:authUrl": "npm config set commercetools-sunrise-data:authUrl",
+    "set:apiUrl": "npm config set commercetools-sunrise-data:apiUrl",
+    "set:projectKey": "npm config set commercetools-sunrise-data:project",
+    "set:clientId": "npm config set commercetools-sunrise-data:client_id",
+    "set:clientSecret": "npm config set commercetools-sunrise-data:client_secret",
+    "get:config": "echo \"authUrl\": && npm config get commercetools-sunrise-data:authUrl && echo \"apiUrl\": && npm config get commercetools-sunrise-data:apiUrl && echo \"projectKey\": && npm config get commercetools-sunrise-data:project && echo \"clientId\": && npm config get commercetools-sunrise-data:client_id && echo \"clientSecret\": && npm config get commercetools-sunrise-data:client_secret",
+    "build": "rimraf dist/ && babel ./**/*.js --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
+    "credentials": "npm run build && node dist/lib/update-credentials.js --project $npm_package_config_project --client_id $npm_package_config_client_id --client_secret $npm_package_config_client_secret --apiUrl $npm_package_config_apiUrl --authUrl $npm_package_config_authUrl",
+    "clean:data": "./mvnw spring-boot:run -DprojectCleanUp=true -DprojectSetUp=false -DchannelsImport=false -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:projectSetup": "./mvnw spring-boot:run -DprojectCleanUp=false -DprojectSetUp=true -DchannelsImport=true -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
```

Comment:
this `projectSetup` task always runs the "inventoryDelete" job which is not necessary to do in the default "start" run.  inventory deletion literally takes ages when you run it on a project that already has inventory and it's not necessary at all because the actual inventory import is now done by our `stock-import` CLI which is not as dumb as the java stuff and actually syncs (= does not fail if the inventory is already there).  So there's no need to delete it all.

Reply:
created a new issue #104